### PR TITLE
coding-partner review

### PR DESCRIPTION
**Pull request by sjdev2212**
In this exercise I: 
Copy all the contents of index.html, style.css, index.js, and interactive.js for my coding partner to perform a code review to check for the correct use of Javascript Html and CSS.

### CODE REVIEW

**Comment by JasemDuncan** [comment]
Hi @sj1978 ! 👌

Your Code is great, clean, and very clear.

There are many things to highlight:

- You used the appropriate declaration of objects using const.
- You used comparison(===) in the best way.

I think it would be better to implement some small things:

- Try to use de declaration on Top.
https://github.com/sj1978/coding-partner-review/blob/94d90d3765867e3c2a1311b9eb85275c7d395c51/index.js#L17

- Try to use more explicit variables.

**Review by JasemDuncan — CHANGES_REQUESTED** [review]
Please consider making the changes listed above.

File: index.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+input.placeholder = 'Add to do...';
+input.id = 'text';
+input.setAttribute('require',true)
+const addButton = document.createElement('button');
+
+addButton.innerText = 'Add';
+addButton.type = 'submit';
+div.append(input, addButton);
+
+const listTodo = new Todos();
```
Comment (sjdev2212) [inline index.js]: good suggestion!

**Review by sjdev2212 — COMMENTED** [review]
Added comment

**Comment by JasemDuncan** [comment]
Thanks for implementing the changes @sj1978, now you can merge your code.
:muscle:

**Review by JasemDuncan — APPROVED** [review]
Thanks for updating your code. Is time to merge.

**Pull request merged**